shared/src/main/scala/drt/shared/Api.scala

```diff
@@ -574,6 +574,10 @@ case class ArrivalUpdate(old: Arrival, updated: Arrival)
 case class ArrivalsDiff(toUpdate: ISortedMap[UniqueArrival, Arrival], toRemove: Set[Arrival])
 
 trait SDateLike {
+  val oneMinuteMillis: Int = 60000
```

Comment:
should these live in an object rather? Considering they are global, and will be the same regardless of the instance you are accessing them on.

Reply:
Yes good point. We have them scattered a little at the mo because of them being used on server side. I'll move all definitions into a shared object to tidy existing uses up